gnuradio-runtime/include/gnuradio/rpcregisterhelpers.h

```diff
@@ -30,6 +30,7 @@
 #include <gnuradio/rpcmanager.h>
 #include <gnuradio/rpcserver_selector.h>
 #include <gnuradio/rpcserver_base.h>
+#include <gnuradio/block_registry.h>
```

Comment:
I am not sure about this. But the `block_registry.h` defined a variable called `global_block_registry` which is used in `rpcregisterhelpers.h`. I did not understand why it was working fine previously.